src/EnergyPlus/HVACManager.cc

```diff
@@ -1619,7 +1619,7 @@ namespace HVACManager {
 				}
 				if ( SimZoneEquipment ) {
 					if ( ( IterAir == 1 ) && ( ! FlowMaxAvailAlreadyReset ) ) { // don't do reset if already done in FirstHVACIteration
-						ResetTerminalUnitFlowLimits();
+						// ResetTerminalUnitFlowLimits(); // don't do reset at all - interferes with convergence and terminal unit flow controls
```

Comment:
Agreed. MaxAvail reset first time through on FirstHVACIteration = true and be done with it.